packages/frontend/src/utils/fontSize.tsx

```diff
@@ -3,6 +3,6 @@ export enum fontSize {
     paragraphMobile = '16px',
     headerDesktop = '32px',
     headerMobile = '30px',
-    subtitleDesktop = '20px',
-    subtitleMobile = '18px',
+    subtitleDesktop = '22px',
+    subtitleMobile = '20px',
```

Comment:
Ta forma FontSize wydaje mi się dzwiwna. Jakie są plusy przypisania rozmiaru do elementu np. do paragrafu desktop 18? Jak header mobile będzie miał 18 to ta wartość wystąpi dwa razy czy w stylowaniu mobile header bedzie miał font-size:${fontSize:paragraphDesktop}?